src/collective/transmute/pipeline/__init__.py

```diff
@@ -1,3 +1,18 @@
+"""
+Pipeline initialization and orchestration for ``collective.transmute``.
+
+This module provides functions and context managers to run, debug, and manage
+pipeline steps for Plone item transformation. Used in the ``collective.transmute``
+pipeline.
+
+Example:
+    .. code-block:: python
```

Comment:
The lexer should in fact be `pycon`, for Python console, where a line begins with the Python interpreter's console prompt of `>>>`. Python code without the prompt uses the `python` lexer. See https://pygments.org/languages/.

Reply:
I reverted these changes in https://github.com/collective/collective.transmute/pull/20/commits/67e12a006960a15eb2ca0589ccd15a6266bf184e.

Reply:
TIL, thanks :-)